src/programs/city/layout.js

```diff
@@ -341,6 +341,8 @@ class CityLayout extends kernel.process {
     const flower2Adjusted = new RoomPosition(flower2Pos.x - LAYOUT_FLOWER_BUFFER, flower2Pos.y - LAYOUT_FLOWER_BUFFER, this.data.room)
     this.planStructureMatrix(layout, flower2Adjusted, LAYOUT_FLOWER, (2 * LAYOUT_FLOWER_BUFFER) + 1)
 
+    this.planRoads(layout, corePos, flower1Pos, flower2Pos)
```

Comment:
Could this go into a separate program? That way it can be applied to the existing rooms.

Reply:
Certainly planning to refactor it into a more widely usable form later that can do multi-room paths.